cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -58,7 +58,7 @@ type describeClusterOptions struct {
 	showOtherConditions string
 	showMachineSets     bool
 	disableNoEcho       bool
-	disableGrouping     bool
+	Grouping            bool
```

Comment:
```suggestion
	grouping            bool
```
The field can remain private.

Reply:
Done with this